test/performance/odp_sched_latency.c

```diff
@@ -858,6 +890,13 @@ static int create_queues(test_globals_t *globals)
 		odp_schedule_group_t grp;
 		int prio;
 
+		if (capa.max_queue_size && queue_size[i] > capa.max_queue_size) {
+			ODPH_ERR("Warn: queues may not be able to store all events (required size "
+				 "%" PRIu32 ", max supported %" PRIu32 ")\n", queue_size[i],
+				 capa.max_queue_size);
+			queue_size[i] = capa.max_queue_size;
+		}
```

Comment:
This could be part of `calc_queue_sizes()` as this is basically just additional queue size calculation.